src/data/packer.json

```diff
@@ -61,25 +61,21 @@
 		{
 			"iconName": "docs",
 			"name": "General Documentation",
-			"mainBranch": "master",
```

Comment:
Packer has switched from `master` to `main`, so this is no longer necessary, and is in fact detrimental.